Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2005-08-23
Initial Package Version: 1.2.2
Upstream Status:         Unknown (discussed on the OpenEXR mailing lists though)
Origin:                  Jeff Clifford (on the OpenEXR mailing list)
                         http://lists.gnu.org/archive/html/openexr-devel/2005-07/msg00024.html
Description:             Fixes GCC-4 related build problem


Index: openexr-1.6.1/exrenvmap/main.cpp
===================================================================
--- openexr-1.6.1.orig/exrenvmap/main.cpp
+++ openexr-1.6.1/exrenvmap/main.cpp
@@ -45,6 +45,7 @@
 #include <iostream>
 #include <exception>
 #include <stdlib.h>
+#include <cstring>
 
 using namespace Imf;
 using namespace std;
Index: openexr-1.6.1/exrmaketiled/main.cpp
===================================================================
--- openexr-1.6.1.orig/exrmaketiled/main.cpp
+++ openexr-1.6.1/exrmaketiled/main.cpp
@@ -45,6 +45,7 @@
 #include <iostream>
 #include <exception>
 #include <string>
+#include <cstring>
 #include <stdlib.h>
 
 using namespace Imf;
